sdk/handleError.js

```diff
@@ -23,18 +23,21 @@ function validateResponse(response) {
     throw new APIError(`Unexpected response ${response.statusCode}`);
   }
 
-  return response.body || response;
+  return response;
 }
 
-async function loggedRequest(options, type = 'post') {
```

Comment:
Because it was an oversight (by me) to believe that validating a request only means you need `response.body`. In my mind it makes much more sense to just return the whole object and let the caller extract as needed.